shopinvader_address/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Shopinvader Address",
+    "summary": """
+        Shopinvader Address: Shipping,Billing""",
+    "version": "16.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV",
+    "website": "https://github.com/shopinvader/odoo-shopinvader",
+    "depends": [
+        "shopinvader_schema_address",
```

Comment:
This should depend on `account` since we search account moves to detect addresses in used in posted invoices.
And also probably on sales to detect addresses used in confirmed sale orders.